CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.0.1 (January 27, 2023)
+
+* Upgrade to Go 1.19.4.
+* Upgrade Envoy Proxy to 1.24.1.
+* Upgrade consul-server-connection-manager to 0.1.1
```

Comment:
Ah, the changelog checker looks in the .changelog directory (similar to how the changelog is managed in Consul).

So you can either just add the pr/no-changelog label to the PR, or you can move one of these to `.changelog/<pr-number>.txt`

Also, should we follow the same style as the Consul changelog with sections for different categories of changes (`IMPROVEMENTS`, `BUG FIXES`, etc).

Reply:
Yah, I am reading through that right now and I see that I messed it up. I shall fix.